.github/workflows/build-and-test.yaml

```diff
@@ -52,11 +52,11 @@ jobs:
 
       - name: Test
         run: make test
+
```

Comment:
We need this in the CI, to make sure submodule matching the generated source codes, https://github.com/apache/skywalking-cli/blob/b3b919d5c89d43aebfc6f25f147bf41db11950bf/.github/workflows/go.yml#L41-L48